bpf/bpf_lxc.c

```diff
@@ -101,6 +101,21 @@ encode_custom_prog_meta(struct __ctx_buff *ctx, int ret, __u32 identity)
 #endif
 
 #ifdef ENABLE_IPV6
+struct ct_buffer6 {
+	struct ipv6_ct_tuple tuple;
+	struct ct_state ct_state;
+	__u32 monitor;
+	int ret;
+};
+
+struct {
+	__uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
+	__type(key, __u32);
+	__type(value, struct ct_buffer6);
+	__uint(pinning, LIBBPF_PIN_BY_NAME);
```

Comment:
Does this need to be pinned? It's not a prog array, so it should follow the normal lifecycle of bpf progs.